src/modules/orders/services/export.service.ts

```
@@ -0,0 +1,83 @@
+            );
+          } catch (error) {
+            this.logger.warn(`Error saving to cache: ${error.message}`);
+          }
+          resolve(filePath);
+        });
+      } catch (error) {
+        clearTimeout(writeTimeout);
+        csvStream.end();
+        reject(error);
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)